### PR TITLE
resolution in WFS handler

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[QueryResolution an WFS übergeben als Vendor-Parameter](https://trello.com/c/FhgOKL1M/152-queryresolution-an-wfs-%C3%BCbergeben-als-vendor-parameter)

File: services/src/handlers/wfs.rs [hunk 1]
```diff
@@ -228,7 +228,9 @@ async fn get_feature<C: Context>(
             let time = TimeInstance::from(chrono::offset::Utc::now());
             TimeInterval::new_unchecked(time, time)
         }),
-        spatial_resolution: SpatialResolution::zero_point_one(),
+        spatial_resolution: request
+            .query_resolution
+            .unwrap_or_else(SpatialResolution::zero_point_one),
```
Comment (michaelmattig) [inline services/src/handlers/wfs.rs:233]: is this is meaningful default value? Should id maybe be dependant of the projection? Maybe put a TODO here to check this

**Review by michaelmattig — CHANGES_REQUESTED** [review]
A question

File: services/src/handlers/wfs.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply services/src/handlers/wfs.rs:234]: f820d8f

**Pull request merged**